src/Controls/src/Core/DateTimeTypeConverter.cs

```diff
@@ -0,0 +1,105 @@
+#if NET6_0_OR_GREATER
+using System;
+using System.ComponentModel;
+using System.Globalization;
+using Microsoft.Maui.Controls.Xaml;
+
+namespace Microsoft.Maui.Controls;
+
+[ProvideCompiled("Microsoft.Maui.Controls.XamlC.DateTimeTypeConverter")]
+public class DateTimeTypeConverter : TypeConverter
```

Comment:
make this internal so we do not change the public api

Reply:
Changed the converter class to internal and removed the related public API changes.